### DISCUSSION TITLE
Recessed door installation

### DISCUSSION THREAD

**Discussion by AlexOwl** [TheRealWaldo/thermal#240] [Q&A]
Hi, is there any common way to process integration to detect movement in-out of room like hiome did before? I want to replicate its functionality since the company is not selling the device anymore and I think the best usage of amg8833 is on top of the door to track the direction. I'm a developer and I can write custom logic for everything and combine it with room-assistant, however I would like to know if there is already existing solution or is it planned to be implemented

**Comment by TheRealWaldo** [comment]
Nothing is planned, but considering how it operates, it would be easy to replicate.  The logic could even go directly into the esphome amg8833 projects I've written that connect to this integration, removing any need for an integration or using room-assistant.

**Reply by AlexOwl** [reply]
do you have support (donate) link?

**Reply by TheRealWaldo** [reply]
I really should create one! lol

Not at the moment.
